spyder/plugins/projects/plugin.py

```diff
@@ -213,18 +215,19 @@ def register(self):
                 lambda v: outline_explorer.update_all_editors())
         if self.main:
             self.sig_project_closed.connect(
-                lambda v: self.main.set_window_title())
+                lambda v, s: self.main.set_window_title())
         if self.completions:
             self.sig_project_closed.connect(
                 functools.partial(self.completions.project_path_update,
                                   update_kind=WorkspaceUpdateKind.DELETION,
                                   instance=self))
         if self.editor:
             self.sig_project_closed.connect(

```

Comment:
```suggestion
            self.sig_project_closed[bool].connect(
```

Here we're using the signal but with its second signature, so we can connect it to `setup_open_files`.